test/basic/ecs_web_create/ecs_web_create_test.go

```diff
@@ -0,0 +1,148 @@
+	ecsServiceAws := terratest_aws.GetEcsService(t, "us-east-1", ecsClusterName, ecsServiceName)
+
+	return ecsComputeAsset, ecsClusterAws, ecsServiceAws, nil
+}
+
+func insecureHttpClient() *http.Client {
+	transport := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+	}
+	client := &http.Client{Transport: transport}
+	return client
+}
+
+func TestECSComputeCreate(t *testing.T) {
+	checkSetup()
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: ".",
+
+		Vars: map[string]interface{}{
+			"organization_id":   os.Getenv("ORGANIZATION_ID"),
+			"environment_id":    os.Getenv("ENVIRONMENT_ID"),
+			"aptible_host":      os.Getenv("APTIBLE_HOST"),
+			"dns_account_id":    os.Getenv("DNS_AWS_ACCOUNT_ID"),
+			"ecs_name":          "ecs-web-test",
+			"container_command": []string{"nginx", "-g", "daemon off;"},
+			"container_image":   "nginx",
+			"container_port":    80,
+			"container_name":    "nginx",
+			"is_public":         true,
+			"is_ecr_image":      false,
+			"vpc_name":          "testecs-web-vpc",
+			"domain":            "aptible-cloud-staging.com",
+			"subdomain":         "test-ecs-integration",
+		},
+	})
+	defer cleanupAndAssert(t, terraformOptions)
+	terraform.InitAndApply(t, terraformOptions)
+
+	c := client.NewClient(
+		true,
+		os.Getenv("APTIBLE_HOST"),
+		os.Getenv("APTIBLE_TOKEN"),
+	)
+	ctx := context.Background()
+
+	aptibleAccountId := terraform.Output(t, terraformOptions, "aptible_aws_account_id")
+	aptibleAccountRole := fmt.Sprintf("arn:aws:iam::%s:role/OrganizationAccountAccessRole", aptibleAccountId)
+	os.Setenv(terratest_aws.AuthAssumeRoleEnvVar, aptibleAccountRole)
+
+	vpcId := terraform.Output(t, terraformOptions, "vpc_id")
+	vpcAsset, vpcAws, err := getAptibleAndAWSVPCs(t, ctx, c, vpcId, "testecs-web-vpc")
+	assert.Nil(t, err)
+	assert.Equal(t, vpcAsset.Id, vpcId)
+	assert.Equal(t, vpcAsset.Status, cac.ASSETSTATUS_DEPLOYED)
+	assert.GreaterOrEqual(t, len(vpcAws), 1)
+	assert.Equal(t, len(vpcAws[0].Subnets), 6)
+
+	ecsWebId := terraform.Output(t, terraformOptions, "ecs_web_id")
+	ecsWebAsset, ecsClusterAws, ecsServiceAws, err := getAptibleAndAWSECSServiceAndCluster(t, ctx, c, ecsWebId, "ecs-web-test-web-cluster", "ecs-web-test")
+	assert.Nil(t, err)
+	assert.Equal(t, ecsWebAsset.Id, ecsWebId)
+	assert.Equal(t, ecsWebAsset.Status, cac.ASSETSTATUS_DEPLOYED)
+	assert.NotNil(t, ecsWebAsset.Outputs)
+	assert.Equal(t, *ecsClusterAws.Status, "ACTIVE")
+	assert.Equal(t, *ecsServiceAws.Status, "ACTIVE")
+
+	ecsLoadBalancerUrl := terraform.Output(t, terraformOptions, "loadbalancer_url")
+	ecsLbGet, ecsLbGetErr := insecureHttpClient().Get(fmt.Sprintf("https://%s", ecsLoadBalancerUrl))
+	assert.Nil(t, ecsLbGetErr)
+	assert.EqualValues(t, ecsLbGet.StatusCode, 200)
+
+	ecsServiceUrl := terraform.Output(t, terraformOptions, "web_url")
+	ecsUrlGet, ecsUrlGetErr := http.Get(fmt.Sprintf("https://%s", ecsServiceUrl))
+	if assert.NoError(t, ecsUrlGetErr) {
```

Comment:
interesting, i always just do two assertions back to back 🤔 (`assert.Nil(t, errorthing`). TIL:

<img width="430" alt="image" src="https://user-images.githubusercontent.com/2961973/200952515-0987c341-8209-45de-b167-2484783aa90b.png">

i usually use testify and/or goconvey and idk if they operate differently, but this lgtm